services/repository/create.go

```diff
@@ -139,24 +139,24 @@ func prepareRepoCommit(ctx context.Context, repo *repo_model.Repository, tmpDir,
 }
 
 // InitRepository initializes README and .gitignore if needed.
-func initRepository(ctx context.Context, repoPath string, u *user_model.User, repo *repo_model.Repository, opts CreateRepoOptions) (err error) {
-	if err = repo_module.CheckInitRepository(ctx, repo.OwnerName, repo.Name, opts.ObjectFormatName); err != nil {
+func initRepository(ctx context.Context, u *user_model.User, repo *repo_model.Repository, opts CreateRepoOptions) (err error) {
+	if err = repo_module.CheckInitRepository(ctx, repo); err != nil {
```

Comment:
It is logic change.

Before: `opts.ObjectFormatName`
Now:  `repo.ObjectFormatName`

Reply:
OK, luckily, there is only one call to initRepository, and `repo.ObjectFormatName` is `opts.ObjectFormatName`

Reply:
Yes, I have checked that.